monai/data/image_reader.py

```diff
@@ -717,7 +722,7 @@ def get_data(
         img,
         location: Tuple[int, int] = (0, 0),
         size: Optional[Tuple[int, int]] = None,
-        level: int = 0,
+        level: Optional[int] = None,
```

Comment:
Could you please help add unit tests for different levels?

Thanks.

Reply:
There are already unit tests for levels 0 and 2. https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_cuimage_reader.py

Reply:
@Nic-Ma, do you think that more is needed or they are fine?

Reply:
OK, looks good to me.

Thanks.